engine/events/treemons.asm

```diff
@@ -51,7 +51,7 @@ RockMonEncounter:
 	xor a
 	ret
 
-	db $05 ; ????
+	db 5 ; unused
```

Comment:
```suggestion
	db 5 ; unreferenced
```
Just a minor nitpick, since it's never referenced. Unused is for data that is referenced but never used.

Reply:
I think the convention is that `; unreferenced` is used for labels. `db ... ; unused` or `dw ... ; unused` are used elsewhere for random bits of unlabeled data. (`git grep -P '^\td[bw] .+ ' un(use|reference)d'`)

Reply:
if the data is part of a referenced table it should be `; unused`, or if its part of a control path never executed. If it is never referenced it should be unreferenced. I see that there are a few marked as `; unused` the `audio/music` files... but i'd argue those are technically wrong.

See the below guidance from the Code Cleanup page.

>Unused/Unreferenced/Stubs
Currently, a mixture of UnusedFunction, Unreferenced_Function and StubbedFunction is being used. This should be changed, where appropriate names can be given, to comments right after the label in question, meaning the following:
>
>`unused` for code or data that is referenced but never used, through branches that are never taken during normal gameplay
>
>`unreferenced` for code or data that's never referenced, thus can never be used, even with glitches, short of arbitrary code execution
stub for functions that were supposed to do something but don't, because their functionality was removed

The above doesn't really make a distinction that a `label` is required. The fact that there *isn't* a label for this `db 5` is simply because we didn't create one, like we did in other `unreferenced` cases.

**_Anyways, i'm only nitpicking, and simply having a conversation on my interpretation. Ultimately i'll support whatever you think is more appropriate in this case_**. 😄

Reply:
Opened #1124 for this, though I expect there would be some discussion about just which comments to update.